agent/containers/images/Makefile

```diff
@@ -48,7 +50,7 @@ URL_PREFIX = https://copr-be.cloud.fedoraproject.org/results/${COPR_USER}
 # the final repo name would be "pbench-test".
 TEST =
 _TEST_SUFFIX = $(if $(TEST),-$(TEST),"")
-_PBENCH_REPO_NAME = pbench${_TEST_SUFFIX}
+_PBENCH_REPO_NAME = pbench-$(shell grep -oE '[0-9]+\.[0-9]+' ${_PBENCH_TOP}/agent/VERSION)${_TEST_SUFFIX}
```

Comment:
I don't like the duplication here with `rpm.mk`, but neither could I figure out a way to resolve it gracefully.

Reply:
I suppose you could unbundle it into a script you `$(shell script {path})` in both places, but I don't think it's that big a deal.

Reply:
I think it is probably best as is.